### ISSUE TITLE
Where do I see a list of all powershell command lets and scripts?  Not just the sample list?

### ISSUE THREAD

**Issue by kmramsey** [Pscx/Pscx#24]
Where can I find a list of all PowerShell command lets and scripts?  Not just the sample list?

Thanks,
Keith

**Comment by rkeithhill** [comment]
The most definitive source is to run `get-command -module pscx`.  You can also man the `about_pscx` help topic.

**Comment by kmramsey** [comment]
That's not possible.  I have to get this thing through a ton of red tape just to download everything and run that command.  So I need to see the possible command so I can justify the hours of paperwork before I even get close to that.  I'm also running machines in offline domains.  I'm kind of locked out of just playing with new code before knowing what's in it.

**Comment by rkeithhill** [comment]
In that case, here is the list of commands:
```

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Alias           Resize-Bitmap                                      3.2.2      pscx
Function        Edit-HostProfile                                   3.2.2      pscx
Function        Edit-Profile                                       3.2.2      pscx
Function        Enable-OpenPowerShellHere                          3.2.2      pscx
Function        Get-ExecutionTime                                  3.2.2      pscx
Function        Get-Parameter                                      3.2.2      pscx
Function        Get-ScreenCss                                      3.2.2      pscx
Function        Get-ScreenHtml                                     3.2.2      pscx
Function        Get-ViewDefinition                                 3.2.2      pscx
Function        help                                               3.2.2      pscx
Function        Import-VisualStudioVars                            3.2.2      pscx
Function        Invoke-BatchFile                                   3.2.2      pscx
Function        Invoke-Elevated                                    3.2.2      pscx
Function        Invoke-GC                                          3.2.2      pscx
Function        Invoke-Method                                      3.2.2      pscx
Function        Invoke-NullCoalescing                              3.2.2      pscx
Function        Invoke-Ternary                                     3.2.2      pscx
Function        less                                               3.2.2      pscx
Function        Out-Speech                                         3.2.2      pscx
Function        QuoteList                                          3.2.2      pscx
Function        QuoteString                                        3.2.2      pscx
Function        Resolve-ErrorRecord                                3.2.2      pscx
Function        Resolve-HResult                                    3.2.2      pscx
Function        Resolve-WindowsError                               3.2.2      pscx
Function        Set-LocationEx                                     3.2.2      pscx
Function        Set-ReadOnly                                       3.2.2      pscx
Function        Set-Writable                                       3.2.2      pscx
Function        Show-Tree                                          3.2.2      pscx
Function        Start-PowerShell                                   3.2.2      pscx
Function        Stop-RemoteProcess                                 3.2.2      pscx
Filter          New-HashObject                                     3.2.2      pscx
Cmdlet          Add-PathVariable                                   3.2.2      pscx
Cmdlet          Clear-MSMQueue                                     3.2.2      pscx
Cmdlet          ConvertFrom-Base64                                 3.2.2      pscx
Cmdlet          ConvertTo-Base64                                   3.2.2      pscx
Cmdlet          ConvertTo-MacOs9LineEnding                         3.2.2      pscx
Cmdlet          ConvertTo-Metric                                   3.2.2      pscx
Cmdlet          ConvertTo-UnixLineEnding                           3.2.2      pscx
Cmdlet          ConvertTo-WindowsLineEnding                        3.2.2      pscx
Cmdlet          Convert-Xml                                        3.2.2      pscx
Cmdlet          Disconnect-TerminalSession                         3.2.2      pscx
Cmdlet          Edit-File                                          3.2.2      pscx
Cmdlet          Expand-Archive                                     3.2.2      pscx
Cmdlet          Export-Bitmap                                      3.2.2      pscx
Cmdlet          Format-Byte                                        3.2.2      pscx
Cmdlet          Format-Hex                                         3.2.2      pscx
Cmdlet          Format-Xml                                         3.2.2      pscx
Cmdlet          Get-ADObject                                       3.2.2      pscx
Cmdlet          Get-AdoConnection                                  3.2.2      pscx
Cmdlet          Get-AdoDataProvider                                3.2.2      pscx
Cmdlet          Get-Clipboard                                      3.2.2      pscx
Cmdlet          Get-DhcpServer                                     3.2.2      pscx
Cmdlet          Get-DomainController                               3.2.2      pscx
Cmdlet          Get-DriveInfo                                      3.2.2      pscx
Cmdlet          Get-EnvironmentBlock                               3.2.2      pscx
Cmdlet          Get-FileTail                                       3.2.2      pscx
Cmdlet          Get-FileVersionInfo                                3.2.2      pscx
Cmdlet          Get-ForegroundWindow                               3.2.2      pscx
Cmdlet          Get-Hash                                           3.2.2      pscx
Cmdlet          Get-HttpResource                                   3.2.2      pscx
Cmdlet          Get-LoremIpsum                                     3.2.2      pscx
Cmdlet          Get-MountPoint                                     3.2.2      pscx
Cmdlet          Get-MSMQueue                                       3.2.2      pscx
Cmdlet          Get-OpticalDriveInfo                               3.2.2      pscx
Cmdlet          Get-PathVariable                                   3.2.2      pscx
Cmdlet          Get-PEHeader                                       3.2.2      pscx
Cmdlet          Get-Privilege                                      3.2.2      pscx
Cmdlet          Get-PSSnapinHelp                                   3.2.2      pscx
Cmdlet          Get-ReparsePoint                                   3.2.2      pscx
Cmdlet          Get-RunningObject                                  3.2.2      pscx
Cmdlet          Get-ShortPath                                      3.2.2      pscx
Cmdlet          Get-TerminalSession                                3.2.2      pscx
Cmdlet          Get-TypeName                                       3.2.2      pscx
Cmdlet          Get-Uptime                                         3.2.2      pscx
Cmdlet          Import-Bitmap                                      3.2.2      pscx
Cmdlet          Invoke-AdoCommand                                  3.2.2      pscx
Cmdlet          Invoke-Apartment                                   3.2.2      pscx
Cmdlet          Join-String                                        3.2.2      pscx
Cmdlet          New-Hardlink                                       3.2.2      pscx
Cmdlet          New-Junction                                       3.2.2      pscx
Cmdlet          New-MSMQueue                                       3.2.2      pscx
Cmdlet          New-Shortcut                                       3.2.2      pscx
Cmdlet          New-Symlink                                        3.2.2      pscx
Cmdlet          Out-Clipboard                                      3.2.2      pscx
Cmdlet          Ping-Host                                          3.2.2      pscx
Cmdlet          Pop-EnvironmentBlock                               3.2.2      pscx
Cmdlet          Push-EnvironmentBlock                              3.2.2      pscx
Cmdlet          Read-Archive                                       3.2.2      pscx
Cmdlet          Receive-MSMQueue                                   3.2.2      pscx
Cmdlet          Remove-MountPoint                                  3.2.2      pscx
Cmdlet          Remove-ReparsePoint                                3.2.2      pscx
Cmdlet          Resolve-Host                                       3.2.2      pscx
Cmdlet          Send-MSMQueue                                      3.2.2      pscx
Cmdlet          Send-SmtpMail                                      3.2.2      pscx
Cmdlet          Set-BitmapSize                                     3.2.2      pscx
Cmdlet          Set-Clipboard                                      3.2.2      pscx
Cmdlet          Set-FileTime                                       3.2.2      pscx
Cmdlet          Set-ForegroundWindow                               3.2.2      pscx
Cmdlet          Set-PathVariable                                   3.2.2      pscx
Cmdlet          Set-Privilege                                      3.2.2      pscx
Cmdlet          Set-VolumeLabel                                    3.2.2      pscx
Cmdlet          Skip-Object                                        3.2.2      pscx
Cmdlet          Split-String                                       3.2.2      pscx
Cmdlet          Stop-TerminalSession                               3.2.2      pscx
Cmdlet          Test-AlternateDataStream                           3.2.2      pscx
Cmdlet          Test-Assembly                                      3.2.2      pscx
Cmdlet          Test-MSMQueue                                      3.2.2      pscx
Cmdlet          Test-Script                                        3.2.2      pscx
Cmdlet          Test-UserGroupMembership                           3.2.2      pscx
Cmdlet          Test-Xml                                           3.2.2      pscx
Cmdlet          Write-BZip2                                        3.2.2      pscx
Cmdlet          Write-Clipboard                                    3.2.2      pscx
Cmdlet          Write-GZip                                         3.2.2      pscx
Cmdlet          Write-Tar                                          3.2.2      pscx
Cmdlet          Write-Zip                                          3.2.2      pscx
```

**Comment by kmramsey** [comment]
Thanks a bunch!

**Comment by i-give-up** [comment]
This issue can be closed because #40 fixed this.

**Comment by rkeithhill** [comment]
Thanks.
